src/modules/crudInvolvementAPI.js

```diff
@@ -1,10 +1,20 @@
 const axios = require('axios').default;
 
 const appId = '19ZJvuqHVwq8iUHpGDJZ';
-const url = `https://us-central1-involvement-api.cloudfunctions.net/capstoneApi/apps/${appId}/likes`;
+const urlLikes = `https://us-central1-involvement-api.cloudfunctions.net/capstoneApi/apps/${appId}/likes`;
 
 export const postPokemonLikeId = (id) => {
-  axios.post(url, { item_id: id });
+  axios.post(urlLikes, { item_id: id });
 };
 
-export default postPokemonLikeId;
\ No newline at end of file
+const urlCommentsPOST = `https://us-central1-involvement-api.cloudfunctions.net/capstoneApi/apps/${appId}/comments`;
+
+export const postPokemonComment = (id, username, comment) => {
+  axios.post(urlCommentsPOST, { item_id: `${id}`, username: `${username}`, comment: `${comment}` });
+};
+
+export const getPokemonComment = (id) => {
+  const urlCommentsGET = `https://us-central1-involvement-api.cloudfunctions.net/capstoneApi/apps/${appId}/comments?item_id=item${id}`;
+  const results = axios.get(urlCommentsGET);
+  return results;
+};
```

Comment:
Nice work posting a comment into the `involvementAPI`, however when I try to watch any Pokemon that don't have a comment, this shows me an error. Maybe you can try to put a `.catch()` into the `axios.get`.